packages/next-rest-framework/src/shared/open-api.ts

```diff
@@ -97,6 +97,13 @@ export interface NrfOasData {
   schemas?: Record<string, OpenAPIV3_1.SchemaObject>;
 }
 
+export const isNrfOasData = (x: unknown): x is NrfOasData => {
+  if (typeof x !== 'object' || x === null) {
+    return false;
+  }
+  return 'paths' in x;
+};
```

Comment:
Could use move this utility to the same file as the CLI script as this is only used by the CLI commands, thanks.

Reply:
ok. I moved it to the CLI script file.